# AG SeqOccIn - 12 Janvier 2021

## Sujet

Avancée du projet depuis AG virtuel été 2020

## Présentation

[Slides](https://docs.google.com/presentation/d/1EG7Kot2PDPTB9JR1YqycvwtIrryX3uRjWh27Ml6n56c/edit#slide=id.gb35a430bc9_0_5)

## Point à aborder

 - SNPs long reads
 - Low coverage SV detection
 - Bovine Benchmark with assembly comparison
 - Zea mays results
 - pipeline

## Proposed outline

### Questions

- How does long read technologies compare with short reads for the detection of variants (SNP as well as SV) ?
- What is the impact of sequence coverage ?
- Can the approaches developed in human be easily applied to other species (bovine and maize) ?
- Variant detection with sparse sequencing
- What is the associated computer cost (running time) ?
- Good practices and pipelines for the detection of variants using short and long reads

### Methods

- Compare long and short reads techno using the giab benchmark in human (SNP and SV)
- Temptative transfer to bovine and Maïze

### Results

- Recall and precision of variant detection with different techno and different sequencing depth, first SNP and then SV in human (particular focus on CLR and it’s - ability to detect variants at different seq depth (SNP, indels and SV))
- Adapting the protocol for the detection of variants in bovine (results)
- Variant detection with sparse sequencing (example of Maïze)
- Computer cost (running time) different techno and species ?
- We provide pipelines for the detection of variants (SNP and SV) using short and long reads and also using assembly comparison



